torch/onnx/utils.py

```diff
@@ -255,18 +274,17 @@ def export(
                         "test.onnx.pb"
                     )
 
-        f: a file-like object (such that ``f.fileno()`` returns a file descriptor)
-            or a string containing a file name.  A binary protocol buffer will be written
-            to this file.
-        export_params (bool, default True): if True, all parameters will
+        f: Path to the output ONNX model file. E.g. "model.onnx".
+        kwargs: Named arguments to the model.
+        export_params: If True, all parameters will
             be exported. Set this to False if you want to export an untrained model.
             In this case, the exported model will first take all of its parameters
             as arguments, with the ordering as specified by ``model.state_dict().values()``
-        verbose (bool, default False): if True, prints a description of the
```

Comment:
Is default removing from doc for formatting?

Reply:
Yes. They are automatically added to the docs from the annotations.